WordPress/gutenberg
# Table Block: Support rowspan attribute in table HTML, including when pasting

Part of #45774
Related to #45981

## What?

This PR adds support for the rowspan attribute to the table block. At the same time, I have taken the rowspan attribute into account when pasting into the editor.

## Why?

In #45981, colspan is now supported.
I think it makes sense to support rowspan as well, since the table may have both merged rows and columns.

## How?

I have made the same changes as were made in #45981.

At the same time, the following two changes are being taken:

- Updated unit test data for `pasteHandler` to include `rowspan`.
- Don't render the `rowspan/colspan` attribute if **the value is 1** (the default value).

## Testing Instructions

- Use HTML or Google Spreadsheet to create cell-merged tables.
- Confirm that the rows and columns remain merged when pasted into the editor.
- In the code editor, confirm that unnecessary `rowspan/colspan` attributes (i.e., **1**) are not added.

https://user-images.githubusercontent.com/54422211/208287038-488f4620-2a80-48ca-b772-b527d57c7bf4.mp4
